integration-tests/src/multichain/containers.rs

```diff
@@ -42,6 +42,8 @@ impl<'a> Node<'a> {
     ) -> anyhow::Result<Node<'a>> {
         tracing::info!("running node container, account_id={}", account_id);
         let (cipher_sk, cipher_pk) = hpke::generate();
+        let sign_sk =
```

Comment:
Why do we need containers again? Should we get rid of it?

Reply:
containers is there if we want to run docker tests. It's not bad to have, but we can evaluate this later